source/jsonnet/northern-ireland/individual/blocks/employment/town_and_county.jsonnet

```diff
@@ -0,0 +1,70 @@
+local placeholders = import '../../../lib/placeholders.libsonnet';
+local rules = import 'rules.libsonnet';
+
+local question(title) = {
+  id: 'town-and-county-question',
```

Comment:
I appreciate this is in the employment folder, I just think when we flatten it to JSON it needs the keyword 'work' in the id's somewhere (work-town-and-county?)

Reply:
I have updated these